### PR TITLE
Fix a few key/value error issues

### PR DESCRIPTION
**Pull request by shifucun**
* These errors are found from Error Reporting in GCP. We should set up alerts for such errors.
* All Flask endpoint should do "try/catch" to handle any errors from mixer response. This PR updates one such case from error reporting logs.
* Changed "dcid" to "dcids" in name fetch endpoint for better readability and consistency.

### CODE REVIEW

File: server/routes/api/place.py [hunk 1]
```diff
@@ -128,9 +128,11 @@
...
 def api_name():
     """Get place names."""
-    dcids = request.args.getlist('dcid')
-    result = names(dcids)
-    return Response(json.dumps(result), 200, mimetype='application/json')
+    dcids = request.args.getlist('dcids')
+    try:
+        return names(dcids)
+    except Exception as e:
+        return '{}'.format(e), 400
```
[the remark this answers is not in the document]
Comment (beets) [reply server/routes/api/place.py:135]: Please fix. We could log the error and return a simplified error message.

File: static/js/utils/place_utils.ts [hunk 2]
```diff
@@ -165,15 +167,16 @@ export function getPlaceNames(
...
+  return axios
+    .get("/api/place/name", {
+      params: {
+        dcids: dcids,
+      },
+      paramsSerializer: stringifyFn,
+    })
+    .then((resp) => {
+      return resp.data;
+    });
```
Comment (beets) [inline static/js/utils/place_utils.ts:179]: Should we add an error handler here, that returns just the dcid's? Could be a TODO as well.

**Review by beets — COMMENTED** [review]
thanks for these fixes! +1 on alerts

File: server/routes/api/place.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (shifucun) [reply server/routes/api/place.py]: Done

File: static/js/utils/place_utils.ts (hunk 2 above)
Comment (shifucun) [reply static/js/utils/place_utils.ts:179]: Yes, we should. I will file a bug to fix all the axios errors handling.

Comment (beets) [reply static/js/utils/place_utils.ts:179]: sounds good - thanks!

**Review by beets — APPROVED** [review]

**Pull request merged**